test/main.tf

```diff
@@ -0,0 +1,8 @@
+provider "aws" {
+  profile = "eng-dev"
```

Comment:
Don't put a `profile` on it. No user will have this profile so they won't be able to run the tests.